README.md

```diff
@@ -76,6 +92,9 @@ See [SDK Sample](./sample) for more examples.
 You may want to remove DeployGate SDK and related code in production build
 to reduce your app's footprint.
 
+In addition, you may want to disable all features provided by DeployGate SDK
+on artifacts for production release.
+
 For your convenience, we provide "Mock" SDK that replaces every function call
 to empty implementation so you don't have to modify your code to switch the builds.
```

Comment:
```suggestion
Do you want to disable DeployGate SDK on production builds? If so, please use `sdk-mock` dependency for production builds instead of `sdk`. `sdk-mock` dependency has public interfaces that are same as of `sdk` but their implementations are empty, so you don't have to modify your app code for specific build variants.
```